libs/examples/src/filter-field/filter-field-infinite-data-depth-example/filter-field-infinite-data-depth-example.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+import { DtFilterFieldDefaultDataSource } from '@dynatrace/barista-components/filter-field';
+
+@Component({
+  selector: 'dt-example-filter-field-infinite-data-depth-filter-field',
+  templateUrl: './filter-field-infinite-data-depth-example.html',
+})
+export class DtExampleFilterFieldInfiniteDataDepth {
+  private DATA = {
+    autocomplete: [
+      {
+        name: 'AUT',
```

Comment:
Since our examples should be understandable for international colleagues as well please change the example to something international e.g. the continent country region city example.